ui/src/widgets/ui-notification/UINotification.vue

```diff
@@ -96,8 +96,12 @@ export default {
                 this.countdown = 100 - (elapsed / parseFloat(this.props.displayTime)) * 100
             }, 100)
         },
-        close () {
+        close (sendMsg) {
```

Comment:
What do you think about making it default to true? (Not sure if Vlue works the same as JS, btw). Than the diff on line 20 can be reverted and it seems cleaner.

```suggestion
        close (sendMsg = true) {
```

Reply:
Well to be honest I have never used default parameters yet in javascript, but if you guys like it then it is fine for me.  Personally I like to see explcit values, because then I immediately see what the value is in each call.  But that is just a matter of personal taste...